Added Windows Sandbox and Windows Host file support

stymiee opened the pull request.

bschwenk commented:
Thank you @stymiee for the contributions!

Couple of questions / comment before pulling this into our wordfiles repo:

1. The Windows Sandbox wordfile contains `XML_LANG` but this may conflict with the XML wordfile.
2. Should the hosts wordfile contain `/Regexp Type = Perl`?
3. The `STYLE_*` declarations are superfluous-- they're used for programming languages only in UEStudio and have no use with other languages
4. The hosts wordfile contains no delimiters -- I'm assuming that's intentional?

stymiee commented:
Thank you @bschwenk for taking the time to review my submission. Your feedback was valuable and I've made adjustments based on them:

1. Removed XML_LANG from the Windows Sandbox wordfile
2. Removed all STYLE_* declarations
3. Corrected omitted delimiters from hosts wordfile. My IDE trimmed off the tab and "space" characters which are the delimiters. I added a pipe at the end since it won't affect anything and make it unlikely that any other program will strip those characters.
4. Added /Regexp Type = Perl to hosts file

bschwenk commented:
Looks good, thanks @stymiee! I'll add these to our Wordfiles manifest as well so that users can download them through the application.
